index.js

```diff
@@ -73,6 +74,25 @@ class Shortcut {
 		 * @type {string}
 		 */
 		this.id = id;
+
+	}
+
+	/**
+	 * Gets the shortcuts metadata from the downloadURL
+	 * @returns {Object} An object with all the metadata about the shortcut
+	 */
+	getMetaData() {
```

Comment:
Metadata is one word, but your camel-casing puts it as meta data. Can you fix that?